client/flutter/ios/fastlane/Appfile

```diff
@@ -0,0 +1,8 @@
+app_identifier("int.who.WHOMyHealth")
+apple_id("") # Your Apple email address
+
+itc_team_id("") # App Store Connect Team ID
+team_id("") # Developer Portal Team ID
```

Comment:
@KrauseFx @joshdholtz - is there any issue with having WHO's `itc_team_id` and `team_id` in a public repo? I see a number of other team ids listed elsewhere in open source code

Reply:
We could use Github Actions secret variables for these.